lib/potluck.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require './lib/dish'
+
+# Potluck Class
+class Potluck
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu = {
```

Comment:
Think about whether or not the class _needs_ this as a state - you could calculate the menu on the fly and slim down your initialize method.